packages/image-resizer/index.js

```diff
@@ -0,0 +1,88 @@
+import { config } from './configs.js'
+import { log } from './utils.js'
+import { Storage } from '@google-cloud/storage'
+import sharp from 'sharp'
+import { join, basename, extname } from 'path'
+import { tmpdir } from 'os'
+import { promises as fs } from 'fs'
+import express from 'express'
+import bodyParser from 'body-parser'
+
+const storage =
+  config.gcs.projectId && config.gcs.keyFilename
+    ? new Storage({
+        projectId: config.gcs.projectId,
+        keyFilename: config.gcs.keyFilename,
+      })
+    : new Storage()
+
+const app = express()
+
+app.use(bodyParser.json())
+app.post('/', async (req, res) => {
+  const event = req.body
+
+  // Extract bucket and file name from protoPayload.resourceName
+  const resourceName = event.protoPayload.resourceName
+  const bucket = resourceName.split('/')[3]
+  const name = resourceName.split('/').slice(5).join('/')
```

Comment:
Suggestion:
check `resourceName` is string or not before invoking `.split` function.